jupyter-tensorflow-cuda-full/rockcraft.yaml

```diff
@@ -0,0 +1,251 @@
+      "${CONDA_BIN}/pip" install --no-cache-dir -r jupyter-requirements.txt
+
+      # Generate a jupyter lab config
+      "${CONDA_BIN}/jupyter" lab --generate-config
+
+      # create Nvidia/CUDA directories
+      mkdir -p "${CRAFT_PART_INSTALL}/usr/local" "${CRAFT_PART_INSTALL}/usr/local/cuda"
+
+      # Install jupyter-tensorflow CUDA
+      # for `cuda-compat-*`: https://docs.nvidia.com/cuda/eula/index.html#attachment-a
+      curl -sL "https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/3bf863cc.pub" | apt-key add - \
+      && echo "deb https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/ /" > /etc/apt/sources.list.d/cuda.list \
+      && apt-get -yq update \
+      && apt-get -yq install --no-install-recommends cuda-compat-${CUDA_VERSION/./-}=${CUDA_COMPAT_VERSION} \
+      cuda-cudart-${CUDA_VERSION/./-}=${CUDA_CUDART_VERSION} \
+      && apt-get clean \
+      && rm -rf /var/lib/apt/lists/*
+
+      # copy Nividia/CUDA directories
+      cp -r /usr/local/cuda-${CUDA_VERSION} ${CRAFT_PART_INSTALL}/usr/local/
+      ln -s ${CRAFT_PART_INSTALL}/usr/local/cuda-${CUDA_VERSION} ${CRAFT_PART_INSTALL}/usr/local/cuda
+
+      # install - other nvidia stuff
+      curl -sL "https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu2004/x86_64/7fa2af80.pub" | apt-key add - \
+      && echo "deb https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu2004/x86_64 /" > /etc/apt/sources.list.d/nvidia-ml.list \
+      && echo "deb https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64 /" > /etc/apt/sources.list.d/nvidia-ml.list \
+      && apt-get -yq update \
+      && apt-get -yq install --no-install-recommends \
+      cm-super cuda-command-line-tools-${CUDA_VERSION/./-} cuda-nvrtc-${CUDA_VERSION/./-} \
+      libcublas-${CUDA_VERSION/./-} libcudnn8=${CUDNN_VERSION}+cuda${CUDA_VERSION} \
+      libcufft-${CUDA_VERSION/./-} libcurand-${CUDA_VERSION/./-} libcusolver-${CUDA_VERSION/./-} \
+      libcusparse-${CUDA_VERSION/./-} libfreetype6-dev libhdf5-serial-dev \
+      libnvinfer7=${LIBNVINFER_VERSION}+cuda${OLD_CUDA_VERSION} \
+      libnvinfer-plugin7=${LIBNVINFER_VERSION}+cuda${OLD_CUDA_VERSION} \
+      libzmq3-dev pkg-config \
+      # can't be used until NVIDIA updates (requires python < 3.7)
+      # python3-libnvinfer=${LIBNVINFER_VERSION}+cuda${CUDA_VERSION} \
+      && apt-get clean \
+      && rm -rf /var/lib/apt/lists/*
+
+      # tensorflow fix - wrong libcuda lib path (+ reconfigure dynamic linker run-time bindings)
+      ln -s ${CRAFT_PART_INSTALL}/usr/local/cuda/lib64/stubs/libcuda.so ${CRAFT_PART_INSTALL}/usr/local/cuda/lib64/stubs/libcuda.so.1 \
+      && echo "/usr/local/cuda/lib64/stubs" > /etc/ld.so.conf.d/z-cuda-stubs.conf \
+      && ldconfig
+
+      # copy CUDA config
+      mkdri -p ${CRAFT_PART_INSTALL}/etc/ld.so.conf.d/
```

Comment:
it's mkdir

Reply:
Not related file got checked in with #35. Updated.